### PR TITLE
Fixes issue redirect to home

### PR DESCRIPTION
**Pull request by Thewhite33**
# Description

I fixed the redirection back to home after redirecting to demo

Fixes # 7

### CODE REVIEW

File: src/components/ui/navbar.tsx [hunk 1]
```diff
@@ -23,7 +23,7 @@ const Navbar = ({
         },
         {
             label: "Demo",
-            link: "/demo",
+            link: "/home#demo",
```
Comment (gargmegham) [inline src/components/ui/navbar.tsx:26]: Just change it to `/home` and I'll merge it in

**Review by gargmegham — CHANGES_REQUESTED** [review]
Just change it to `/home` and I'll merge it in

**Comment by Thewhite33** [comment]
If I give only /home then demo page will not work so I chosed /home#demo

**Comment by gargmegham** [comment]
> If I give only /home then demo page will not work so I chosed /home#demo

Your fix is not working, check out deployed preview https://vercel.live/open-feedback/shield-peer-git-fork-thewhite33-fixes-7-redi-04f3d2-megham-garg.vercel.app?via=pr-comment-visit-preview-link&passThrough=1

**Comment by gargmegham** [comment]
Before raising pull requests, please test your changes on local.

Do read the contributing guidelines and follow `.github/PULL_REQUEST_TEMPLATE.md` for PR description

**Comment by Thewhite33** [comment]
I checked locally it worked fine there but it's not working now I will update this.
Apologies

**Review by gargmegham — APPROVED** [review]
LGTM
Screenshot 2024-06-21 at 7 48 10 PM

**Pull request merged**